### PR TITLE
Exericse1.1.4

### PR DESCRIPTION
**Pull request by ashpace**
Please approve my branch

### CODE REVIEW

File: E1_1_4.m [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+% define input parameters
+n = 10000;
+meu_t=0;
+sigma2_t=.1;
+df_t=8;  % NOTE: see how the final results change if you increase nu (=4 is enough)
+
+meu_L=.1;
+sigma2_L=.2;
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate Student sample with above parameters
```
Comment (GiuseppeMascolo) [inline E1_1_4.m]: this comment isn't complete you are also generating a Lognormal

File: E1_1_4.m [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+z = -1: 0.01 : 3;
```
Comment (GiuseppeMascolo) [inline E1_1_4.m]: why between -1 and 3?

File: E1_1_4.m [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+pdfY = pdf('Lognormal', z, meu_L, sqrt(sigma2_L)); 
+pdfZ = pdfX + pdfY; 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate plot of adding the pdf's
+
+figure
+plot(z,pdfZ)
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% generate plot of samples
+pdX = makedist('tLocationScale', 0, sqrt(0.1), 8);
```
Comment (GiuseppeMascolo) [inline E1_1_4.m]: you are creating again a tStudent distribution, why don't use the same?

File: E1_1_4.m [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+UniformSample = rand(n,1);
+XSample = icdf(pdX,UniformSample);
+YSample = icdf(pdY,UniformSample);
+ZSample = XSample + YSample;
+
+w = 1/n;
+scale = w:w:1;
+
+figure
+plot(ZSample, scale, '.');
```
Comment (GiuseppeMascolo) [inline E1_1_4.m]: why are you plotting the Zsample in scale belonging to the interval [1/n,1]??

**Review by GiuseppeMascolo — CHANGES_REQUESTED** [review]
You need to ad the empirical distribution and empirical quantile.
Great job!

File: E1_1_4.m (hunk 4 above)
Comment (ashpace) [reply E1_1_4.m]: Good question

**Review by ashpace — COMMENTED** [review]
The comments are fair and will be addressed shortly

**Comment by ashpace** [comment]
@GiuseppeMascolo your concerns are addressed. Please review for approval. X Ash

**Pull request merged**